### PR TITLE
Lowpop Antag: Burglar

### PR DESCRIPTION
**Pull request by Geevies**
* Added a lowpop gamemode called 'Burglars', featuring a new type of antagonist, a mix between a heister and a traitor. Neutered for lowpop fun.
* Added a mixed gamemode, 'Conflict', which is heisters and burglars.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/antagonist/outsider/burglar.dm [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+		var/obj/item/clothing/under/uniform = player.w_uniform
+		if(istype(uniform) && uniform.can_attach_accessory(holster))
+			uniform.attackby(holster, player)
+		else
+			player.put_in_any_hand_if_possible(holster)
+
+/datum/antagonist/burglar/proc/equip_ammo(var/mob/living/carbon/human/player, var/obj/item/gun/gun)
+	if(istype(gun, /obj/item/gun/projectile))
+		var/obj/item/gun/projectile/bullet_thrower = gun
+		if(bullet_thrower.magazine_type)
```
Comment (Geevies) [inline code/game/antagonist/outsider/burglar.dm:95]: note to self, spawn these into the briefcase

File: code/game/antagonist/outsider/burglar.dm [hunk 2]
```diff
@@ -0,0 +1,107 @@
+var/datum/antagonist/burglar/burglars
+
+/datum/antagonist/burglar
+	id = MODE_BURGLAR
+	role_text = "Burglar"
+	role_text_plural = "Burglars"
+	bantype = "burglar"
+	antag_indicator = "mutineer"
+	landmark_id = "burglarspawn"
+	welcome_text = "Use :H to talk on your encrypted channel."
```
Comment (Geevies) [inline code/game/antagonist/outsider/burglar.dm:10]: add onto this to make it slightly more fun

File: code/game/antagonist/antagonist.dm [hunk 3]
```diff
@@ -112,7 +112,7 @@
 			log_debug("[key_name(player)] is not eligible to become a [role_text]: They are blacklisted for this role!")
 		else if(player_is_antag(player))
 			log_debug("[key_name(player)] is not eligible to become a [role_text]: They are already an antagonist!")
-		else if(establish_db_connection(dbcon) && required_age && required_age > player.current.client?.player_age)
+		else if(dbcon?.IsConnected() && required_age && required_age > player.current.client?.player_age)
```
Comment (skull132) [inline code/game/antagonist/antagonist.dm:115]: Please don't change this. The logic may not be equivalent if establish_db_connection gets changed.

Comment (Geevies) [inline code/game/antagonist/antagonist.dm:115]:
```suggestion
		else if(establish_db_connection(dbcon) && required_age && required_age > player.current.client?.player_age)
```

**Pull request merged**